src/Components/Admins/FormAdmins/index.js

```diff
@@ -98,15 +99,7 @@ const FormAdmins = () => {
       </div>
       <form onSubmit={onSubmit} className={styles.form__admins}>
         <div>
-          <label>Name</label>
-          <input
-            className={styles.input}
-            type="text"
-            name="name"
-            value={adminInput.name}
-            onChange={onChange}
-            required
-          />
+          <Input type={'text'} name={'Name'} value={adminInput.name} onChange={onChange} required />
```

Comment:
```suggestion
          <Input type={'text'} name={'Name'} value={adminInput.name} onChange={onChange} />
```
You can delete this required because it has been passed inside the input component

Reply:
thanks for the comment